keyboards/kb_elmo/3liza/readme.md

```diff
@@ -0,0 +1,28 @@
+# 3liza
+
+![3liza](https://i.imgur.com/OGpPhRel.jpeg)
```

Comment:
```suggestion
![3liza](https://i.imgur.com/OGpPhRelh.jpeg)
```

Reply:
It's already thumbailed, you can change the `l` to an `h` if you want (max 1024px across).